src/Playwright.TestAdapter/PlaywrightSettingsXml.cs

```diff
@@ -57,6 +57,10 @@ public PlaywrightSettingsXml(XmlReader reader)
                         reader.Read();
                         ExpectTimeout = float.Parse(reader.Value);
```

Comment:
This will use the `CurrentCulture` is that intentional?

Reply:
seems unrelated to that PR do you mind filing an issue with more details or are you interested in contributing a fix to it?